metron-platform/metron-parsing/metron-parsing-storm/src/main/java/org/apache/metron/parsers/bolt/ParserBolt.java

```diff
@@ -326,6 +329,14 @@ protected void handleError(String sensorType, byte[] originalMessage, Tuple tupl
     ErrorUtils.handleError(collector, error);
   }
 
+  protected List<String> getMessageIds(int size) {
+    List<String> messageIds = new ArrayList<>();
+    for(int i =0; i < size; i++) {
+      messageIds.add(UUID.randomUUID().toString());
```

Comment:
Why do we have to generate the UUID, instead of just using the message's GUID?  I thought everything coming out of the `ParserRunner` has to have a GUID.

Reply:
I checked and you are correct.  I will change it to use the GUID instead.  Is it worth generating the UUID in case the GUID is missing for some reason?

Reply:
This is done.